modules/pages/partials/docker.adoc

```diff
@@ -0,0 +1,8 @@
+This SDK runs on top of the {cpp} core, {cbpp}, which requires a {cpp} 17 compiler, such as GCC 8.0, or more recent versions.
+Centos 7 ships with an older version of GCC.
+
+Workarounds are available for installing a newer build chain with https://docs.fedoraproject.org/en-US/epel/[EPEL], 
+but this may not be within your company's policy, so also take a look at container options -- 
+such as the unofficial Docker builds https://github.com/raycardillo/couchbase-dockerfiles[here], 
+which can at least be used as a reference to a known working set-up, 
+to attempt a version specific to your circumstances, and any subsequent change in dependencies.
```

Comment:
Hmm, I've added a few dockerfiles to sdk repos, without flagging them as "official" (but I wonder if their mere presence in the repo implies it?)

Might be worth seeing if @raycardillo wants to fork that repo to couchbaselabs? It could still be marked unofficial, but best-endeavours, and at least it's *supplied* by Couchbase, so known to be in-good-faith as a good starting point?